apiserver/facades/controller/firewaller/firewaller.go

```diff
@@ -307,26 +307,6 @@ func (f *FirewallerAPI) WatchIngressAddressesForRelations(relations params.Entit
 	return results, nil
 }
 
-// MacaroonForRelations returns the macaroon for the specified relations.
-func (f *FirewallerAPI) MacaroonForRelations(args params.Entities) (params.MacaroonResults, error) {
```

Comment:
I think this is a different type of macaroon, not for charm store. It's related to CMR, which wouldn't be contacting the charm store. Should investigate this more.